src/driver/compile.ml

```diff
@@ -278,8 +278,11 @@ let html_generate ~occurrence_file ~remaps ~generate_json output_dir linked =
       let sidebar =
         match sidebar with
         | None -> None
-        | Some { output_file; json } ->
+        | Some { output_file; json; pkg_dir } ->
             Odoc.sidebar_generate ~output_file ~json index.output_file ();
+            Odoc.sidebar_generate
+              ~output_file:Fpath.(output_dir // pkg_dir / "sidebar.json")
+              ~json:true index.output_file ();
```

Comment:
Where is this json file used ? I find this code a bit brittle because if this file is ever needed somewhere else, nothing will ensure that it continues to be generated.
Any way the caller could set `json = true` ?

Reply:
It's used by ocaml.org - this is the sidebar json that served up for the per-package sidebar. The ocaml.org server reads this and renders its own sidebar.